src/main/java/service/GameCounter.java

```diff
@@ -0,0 +1,23 @@
+package service;
```

Comment:
We should create some base package for project

Reply:
- [x] creted base package for project